scene/resources/default_theme/default_theme.cpp

```diff
@@ -30,14 +30,13 @@
 
 #include "default_theme.h"
 
+#include "default_font.gen.h"
+#include "scene/resources/font.h"
 #include "scene/resources/theme.h"
 
 #include "core/os/os.h"
 #include "theme_data.h"
 
-#include "font_hidpi.inc"
-#include "font_lodpi.inc"
-
 #include "servers/text_server.h"
```

Comment:
While at it I'd bring those headers back in the same block, unless there's a reason to have them in a specific order.